quinn-udp/src/lib.rs

```diff
@@ -68,6 +68,7 @@ pub struct RecvMeta {
     pub ecn: Option<EcnCodepoint>,
     /// The destination IP address which was encoded in this datagram
     pub dst_ip: Option<IpAddr>,
+    pub gso_size: Option<usize>,
```

Comment:
Should this be `gro_size`? Or maybe `stride`?

Does this need to be `Option`, or could we default it to be equal to `len` to make things simpler for the caller?

Reply:
I went for stride and made it default to len